config_files/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: build gen-measurements
+
+build:
```

Comment:
Does it make sense to have this in `config_files`?

Reply:
I had it first for each of the products separately. But I noticed that they all have the same Makefile as a common and use it. So I put it outside instead.
I can create a directory called common and put it in there instead. 🤔